docs/iris/example_code/Meteorology/wind_speed.py

```diff
@@ -0,0 +1,69 @@
+"""
+Plotting wind direction using quiver
+===========================================================
+
+This example demonstrates using quiver to plot wind speed
+
+The second plot uses normalised data to produce arrows with a uniform size
+
+"""
+
+import matplotlib.pyplot as plt
+import numpy as np
+
+import iris
+import iris.coord_categorisation
+import iris.plot as iplt
```

Comment:
You only use `iplt` to show your plots. You could remove this import entirely and use `qplt.show` instead to show your plots.